docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
+services:
+  rstudio:
+    build:
+      context: ./
+      dockerfile: Dockerfile-dev
+      args:
+        rstudioPort: 8787
+        rVersion: 3.6.2
+    container_name: iatlas-rstudio
+    ports:
+      - 8787:8787
+    volumes:
+      - .:/home/rstudio
+    networks:
```

Comment:
I'm not sure we actually need this:
```
networks:
   - postgres
```

Try removing it and see if it still works.